app/package.json

```diff
@@ -23,7 +23,8 @@
     "test-chrome-headless": "yarn test && node node_modules/.bin/mocha-headless-chrome -f tests/index.html -a disable-web-security -a user-data-dir=tests/.chrome -a disable-site-isolation-trials"
   },
   "dependencies": {
-    "@cardano-foundation/ledgerjs-hw-app-cardano": "^2.2.1",
+    "@cardano-foundation/ledgerjs-hw-app-cardano": "https://github.com/vacuumlabs/ledgerjs-cardano-shelley/releases/download/v3.0.0-rc.2/cardano-foundation-ledgerjs-hw-app-cardano-v3.0.0-rc.2.tgz",
+    "@emurgo/js-chain-libs": "https://github.com/SebastienGllmt/js-chain-libs-pkg",
```

Comment:
Do you need this for anything? These are the Jormungandr bindings

Reply:
I have no idea how that got back in there, not intentionally for sure.. :D , Thnx for noticing :+1: